File: Java/Arrays/SumOfElements.java
public class SumOfElements {
    public static void main(String[] args) {
        int[] arr = { 1, 2, 3, 4, 5 };
        System.out.println("Sum of Elements in Array: " + sumofElements(arr));
    }

    public static int sumofElements(int[] arr) {
        int sum = 0;

        for (int i = 0; i < arr.length; i++) {
            sum += arr[i];
        }
        return sum;
    }
}

File: C++/Arrays/sum-Of-Elements.cpp
#include <bits/stdc++.h>
using namespace std;

/*
int solve(int arr[], int n){
    int sum =0; 
    for(int i =0; i<n; i++){
        sum += arr[i];
    }
    return sum;
}
*/

int solve(int arr[], int n){
    int initialSum =0;
    int sum = accumulate(arr, arr+n, initialSum);
    return sum;
}

int main() {
    int n;
    cin>>n;

    int arr[n];

    for(int i =0; i<n; i++){
        cin>>arr[i];
    }

    int ans = solve(arr, n);
    cout<<ans<<endl;

    return 0;
}